cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -101,6 +92,26 @@
   mode '0755'
 end
 
+template "#{node['cluster']['scripts_dir']}/slurm/slurm_fleet_manager" do
+  source 'slurm/fleet_manager_program.erb'
+  owner node['cluster']['slurm']['user']
+  group node['cluster']['slurm']['group']
+  mode '0744'
+end
```

Comment:
shall we install the script at install time? following the same approach used for the daemons

Reply:
since this is slurm specific, I have followed the same approach done for resume and suspend program

Reply:
Ok probably we should move everything to install time but we can leave as is for now